magpie-aws/src/main/java/io/openraven/magpie/plugins/aws/discovery/services/RDSDiscovery.java

```diff
@@ -361,51 +362,70 @@ private Map<String, Double> formatDataMapSum(List<Datapoint> map) {
     return datapointMetrics;
   }
 
-  private Map<String, Object> getRDSCloudWatchMetrics(String identifier, MagpieAwsResource data, Logger logger, MagpieAWSClientCreator clientCreator, boolean cluster) {
+  private Map<String, Object> getRDSCloudWatchMetrics(String identifier, String engine, MagpieAwsResource data, Logger logger, MagpieAWSClientCreator clientCreator) {
+    String readMetric;
+    String writeMetric;
     Map<String, Object> requestMetrics = new HashMap<>();
     List<Dimension> dimensions = new ArrayList<>();
 
-    if (cluster) {
+    if (engine.equalsIgnoreCase("aurora-mysql")) {
```

Comment:
Not a blocker, but just FYI if there's any chance that `engine` could be null, it's safer to reverse this so that it reads `"aurora-mysql".equalsIgnoreCase(engine)`.  This version is guaranteed to never throw an NPE.

Reply:
Cheers Jason, quickly addressed this.